### DISCUSSION TITLE
My achievements are not updating! (Pair Extraordinaire)

### DISCUSSION THREAD

**Discussion by gonzalo-cordova-pou** [community/community#40386] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have the basic Achievement for Pair Extraordinaire, but its not detecting my new co-author commits. My colleague has got the upgraded achievement thanks to co-authoring with me! So strange...

**This is my achievement**:

![image](https://user-images.githubusercontent.com/71346949/204638761-fe6547c8-b485-4a1b-96db-c72d18891ef6.png)

**This is my colleague achievement (note that he got the x2 by co-authoring with me!):**

![image](https://user-images.githubusercontent.com/71346949/204639187-ec2c5da4-46a8-4db4-97e1-6aef8706e3dd.png)

**Comment by SilasRodrigues19** [comment]
@gonzalo-cordova-pou Hi.

If he places you as a co-author, it will only be counted for him and the opposite also happens.

**Comment by LeleDallas** [comment]
I'm facing the same issue... I was `co-author` for more than 20 commits and my badge isn't updating... Any solution?

**Comment by Hohrandrey** [comment]
It`s just a bug

**Reply by LeleDallas** [reply]
I opened a ticket with GitHub Support, and they unlocked the achievement after I provided them with all the commits where I was a co-author
